enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/log/segmented/SegmentFile.java

```diff
@@ -262,4 +262,17 @@ public long size()
     {
         return fileSystem.getFileSize( file );
     }
+
+    @Override
+    public void close() throws DisposedException
+    {
+        if ( closed )
+        {
+            throw new RuntimeException( CLOSED_ERROR_MESSAGE );
```

Comment:
Logging which file it is that got the error might be useful when debugging later.